core/src/main/java/com/scalar/db/sql/DdlStatementExecutor.java

```diff
@@ -125,6 +142,17 @@ public Void visit(DropNamespaceStatement statement, Void context) {
   public Void visit(DropTableStatement statement, Void context) {
     try {
       admin.dropTable(statement.namespaceName, statement.tableName, statement.ifExists);
+
+      // Invalidate the metadata cache
+      metadata
```

Comment:
Should it be done in the finally clause to care for the case where dropTable fails for some reason?
(it's a minor case, but if `dropTable` fails after dropping the specified table, the corresponding metadata will be left un-invalidated?)

Reply:
Thank you for taking a look at this! Yes, that's better. I will modify it in that way. Thanks.